snooty/parser.py

```diff
@@ -69,7 +69,6 @@
 from .gizaparser.nodes import GizaCategory
 from .icon_names import ICON_SET
 from .n import FileId, SerializableType, TocTreeDirectiveEntry
-from .openapi import OpenAPI
```

Comment:
We could probably delete `openapi.py` as well. I don't think we reference any of its related classes or methods anywhere else

Reply:
good catch! was looking for any other imports. removed